public/utils/utils.tsx

```diff
@@ -163,3 +165,17 @@ export const getTitleWithCount = (title: string, count: number | string) => {
     </EuiTitle>
   );
 };
+
+export function convertTimestampToString(timestamp: number | string) {
+  if (typeof timestamp === 'string') {
+    return timestamp;
+  }
+  return moment(timestamp).format();
```

Comment:
Should we add date format here ? Like "MM/DD/YYYY hh:mm A"

Reply:
Good catch, will add the enforced output format.

Reply:
Fixed in latest commit.